frontend/tests/controllers-test/dashCtrl.test.js

```diff
@@ -182,7 +182,7 @@ describe('Unit tests for dashboard controller', function () {
                 },
             ];
             vm = createController();
-            expect(vm.hostTeamCount).toEqual(successResponse.count);
+            expect(vm.hostTeamCount).toEqual(successResponse.results.length);
```

Comment:
This is your issue. Change this to successResponse.length

Reply:
There is no property of `results` in the `successResponse` array

Reply:
@nsjcorps when i had done console log of "response.data" in dadhCtrl,results property was giving me the list of the array containing the data.